sdks/python/apache_beam/dataframe/io.py

```diff
@@ -58,6 +58,17 @@
 _DEFAULT_BYTES_CHUNKSIZE = 1 << 20
 
 
+@frame_base.with_docs_from(pd)
```

Comment:
I'm not sure it makes sense to pull the docs from pandas in this case. We're diverging from pandas pretty significantly here: there are a lot of arguments we don't need (e.g. auth related ones), and we prefer a `table` argument rather than `query`. I'd suggest writing a whole new docstring (possibly drawing heavily from the pandas one).

Reply:
Sure-- for clarity here, should the new docstring be documenting the args to read_gbq, as well as the ReadGbq PTransform (e.g. some of the comments written in the ReadGbq PTransform documentation)?

Reply:
Yeah, the new docstring here should be all of the user-facing documentation. _ReadGbq doesn't need to have a docstring since it's a private implementation detail (it could still have one to document its purpose, but this is for Beam devs, not for users).